### ISSUE TITLE
winit: key released before physical release on non special keys

### ISSUE THREAD

**Issue by FloVanGH** [slint-ui/slint#3450]
With the `winit` backend `key-released` is triggered directly after `key-pressed` and before the physical release of the key. This happens only by keys that are not part of the `Key` enum. 

### Example

```slint
export component MainWindow inherits Window {
    title: "gb_runner";

    forward-focus: i-focus-scope;

    i-focus-scope := FocusScope {
        key-pressed(event) => {
            debug("pressed \{event.text}");
            accept
        }

        key-released(event) => {
            debug("released \{event.text}");
            accept
        }
    }
}
```

```rust
use slint::*;

slint::include_modules!();

fn main() -> Result<(), PlatformError> {
    let main_window = MainWindow::new()?;
    main_window.run()
}
```

The output of this with pressed `a` key on the  `winit` backend is:

```sh
pressed a
release a
```

The output of this with pressed `a` key on the  `qt` backend is:

```sh
pressed a
```

**Comment by ogoffart** [comment]
I think this will be fixed with winit 0.29  => https://github.com/slint-ui/slint/pull/3447
